meta-lxatac-software/recipes-support/imx-uuu/imx-uuu_1.5.201.bb

```diff
@@ -2,15 +2,16 @@ SUMMARY = "Universal Update Utility"
 DESCRIPTION = "Image deployment tool for i.MX chips"
 HOMEPAGE = "https://github.com/NXPmicro/mfgtools"
 
-SRC_URI = "https://github.com/nxp-imx/mfgtools/releases/download/uuu_${PV}/uuu_source-uuu_${PV}.tar.gz"
-SRC_URI[sha256sum] = "c763b87ffdf10ac5499a0c319463759caa336bc6567b56d6d0ef448590c1a76d"
+SRCBRANCH = "master"
+SRCREV = "727fc2b782156419a167ebfc8d9a4204ad2eedbf"
+SRC_URI = "git://github.com/nxp-imx/mfgtools.git;protocol=https;branch=${SRCBRANCH}"
```

Comment:
Fine for me if the intention is to align with the meta-freescale recipe.

https://git.yoctoproject.org/meta-freescale/tree/recipes-devtools/uuu/uuu_git.bb?h=master

`SRCREV` should not be placed before `SRC_URI` and `SRCBRANCH` is not required.